tests/cpp/test_resize.cpp

```diff
@@ -89,11 +83,7 @@ TEST_P(ResizeTest, Pad1) {
   std::vector<c10::IValue> aten_inputs({t0});
 
   EnableOptionsGuard enable_options_guard;
-  if (GetParam()) {
-    EnableOptionsGuard::getCurOptions().set(EnableOption::IdModel, {"all"});
-  } else {
-    EnableOptionsGuard::getCurOptions().unset(EnableOption::IdModel);
-  }
+  EnableOptionsGuard::getCurOptions().set(EnableOption::IdModel, {"all"});
```

Comment:
I still keep these options here because "all" also enables loop generation using IdModel, which is required for inlining. I should probably also enable that in lower2device.cpp, but that'll be a future PR.